src/pages/admin-ui-sdk/configuration.md

```diff
@@ -106,7 +112,9 @@ You can download a sample app from the [Commerce UI test extension repository](h
     }).listen(9090);
     ```
 
-1.  Run the local server:
+1. Generate the key.pem and cert.pem certificates in the same directory. You can use `openssl` command.
```

Comment:
It would be good to add ready-to-use example of how to generate keys with openssl command.